src/main/webapp/app/entities/participation/participation.component.html

```diff
@@ -69,7 +69,9 @@ <h2>
                     <td>{{ 'artemisApp.InitializationState.' + participation.initializationState | translate }}</td>
                     <td>{{ participation.initializationDate | date: 'medium' }}</td>
                     <td>
-                        <a routerLink="/user-management/{{ participation.student?.login }}/view">{{ participation.student?.firstName }} {{ participation.student?.lastName }}</a>
+                        <a routerLink="/admin/user-management/{{ participation.student?.login }}/view"
```

Comment:
The route was wrong and causes the following error:
https://sentry.io/share/issue/753d8d6f88004ed4b1c8f9a37d0c5b8e/

Reply:
Sentry link is now outdated :(